node/modules/lp2p/pubsub.go

```diff
@@ -28,12 +30,37 @@ func init() {
 	pubsub.GossipSubDirectConnectInitialDelay = 30 * time.Second
 }
 
-func GossipSub(mctx helpers.MetricsCtx, lc fx.Lifecycle, host host.Host, nn dtypes.NetworkName, bp dtypes.BootstrapPeers, cfg *config.Pubsub) (service *pubsub.PubSub, err error) {
+type GossipIn struct {
+	fx.In
+	Mctx helpers.MetricsCtx
+	Lc   fx.Lifecycle
+	Host host.Host
+	Nn   dtypes.NetworkName
+	Bp   dtypes.BootstrapPeers
+	Cfg  *config.Pubsub
+}
+
+func getDrandTopic() (string, error) {
+	var drandInfo = struct {
+		Hash string `json:"hash"`
+	}{}
+	err := json.Unmarshal([]byte(build.DrandChain), &drandInfo)
+	if err != nil {
+		return "", xerrors.Errorf("could not unmarshal drand chain info: %w", err)
+	}
+	return "/drand/pubsub/v0.0.0/" + drandInfo.Hash, nil
+}
+
+func GossipSub(in GossipIn) (service *pubsub.PubSub, err error) {
 	bootstrappers := make(map[peer.ID]struct{})
-	for _, pi := range bp {
+	for _, pi := range in.Bp {
 		bootstrappers[pi.ID] = struct{}{}
 	}
```

Comment:
sorry I meant this list -- the drand bootstrappers must be included so that we can assign a sufficient high score (over 1000) to perform PX.